src/conversion/view-selection-to-model-converters.js

```diff
@@ -39,10 +39,8 @@ export function convertSelectionChange( modelDocument, mapper ) {
 
 		modelSelection.setRanges( ranges, viewSelection.isBackward );
 
-		if ( !modelSelection.isEqual( modelDocument.selection ) ) {
```

Comment:
What we really want to achieve is to re-render selection. So I think that what @pjasiun suggested will be fine. The problem is that we don't have a reference to `view.Document` there :(.

Anyway, as I said, what we want to achieve is re-rendering and we do so through `enqueueChanges`. Since nothing changes in `model.Selection` no event will be fired by it.

But we could also just fire `changesDone` event (but it's a bit hacky).

Or, pass the reference there somehow.